New version: Zint.Zint version 2.15.0

DandelionSprout opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/267423)

DandelionSprout committed:
New version: Zint.Zint version 2.15.0

DandelionSprout committed:
Added "MinimumOSVersion".

DandelionSprout committed:
Added more rows.

DandelionSprout commented:
Note that most of the "Missing Properties value" errors (e.g. InstallerLocale, InstallScope, etc.) in the pipeline involve things that are irrelevant to v2.15.0, which is a portable program, whereas 2.4.2.0 was an installable program.

stephengillie commented:
Automatic Validation ended with:
>  Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\zint.exe returned exit code: -1073741515
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | C0000135 | -1073741515 | 3221225781 | STATUS_DLL_NOT_FOUND | Possibly due to a missing dependency. (The most common dependency is `Microsoft.VCRedist.2015+.x64` / `.x86`) | 


(Automated response - build 1075.)

ItzLevvie commented:
@wingetbot run

DandelionSprout committed:
I *think* this is the right dependency?

It was rather difficult to figure out from Zint's GitHub repo, but their files mentioned Visual Studio often.

DandelionSprout committed:
To check 1 more off the "Missing Properties" notes.

DandelionSprout commented on manifests/z/Zint/Zint/2.15.0/Zint.Zint.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+ArchiveBinariesDependOnPath: true
+Installers:
+- Architecture: x86
+  InstallerUrl: https://deac-riga.dl.sourceforge.net/project/zint/zint/2.15.0/zint-2.15.0-win32.zip
```
![image](https://github.com/user-attachments/assets/6097de04-0dcc-40a5-84da-521e8180fc6d)

UnownPlain replied:
I'm able to update this package with komac and YamlCreate. Maybe this is a bug in Wingetcreate?

ItzLevvie commented:
[Policy] Reset Feedback
[Policy] Reset Labels
